xml/System.Windows/FrameworkElement.xml

```diff
@@ -1353,17 +1353,17 @@
+ The following example illustrates the dependency property metadata override usage discussed in Remarks. This code defines a custom control class `NumericUpDown` intended to be used from a dedicated control library assembly. The illustrated static constructor references some private initialization function, registers a class handler (another common control subclassing scenario; see [Marking Routed Events as Handled, and Class Handling](/dotnet/framework/wpf/advanced/marking-routed-events-as-handled-and-class-handling)) and finally overrides the <xref:System.Windows.FrameworkElement.DefaultStyleKey%2A> dependency property metadata on the `NumericUpDown` class. <xref:System.Windows.FrameworkElement.DefaultStyleKey%2A> always returns its own type as the intended key, which is the convention that the theme style system uses to look up the style for some arbitrary otherwise non-styled control. The complete sample also defines the actual control's theme style that is referenced by that key; see [NumericUpDown Custom Control with Theme and UI Automation Support Sample](https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp).  
   
  :::code language="csharp" source="~/snippets/csharp/System.Windows/FrameworkElement/DefaultStyleKey/NumericUpDown2.cs":::
  :::code language="vb" source="~/snippets/visualbasic/VS_Snippets_Wpf/CustomControlNumericUpDown/visualbasic/customcontrollibrary/numericupdown2.vb":::  
 
-  The complete source code for this example is available on GitHub for both [C#](https://github.com/dotnet/dotnet-api-docs/tree/main/snippets/csharp/VS_Snippets_Wpf/CustomControlNumericUpDown/CSharp/CustomControlLibrary) and [Visual Basic](https://github.com/dotnet/dotnet-api-docs/tree/main/snippets/visualbasic/VS_Snippets_Wpf/CustomControlNumericUpDown/visualbasic/customcontrollibrary).
+  The complete source code for this example is available on GitHub for both [C#](https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp) and [Visual Basic](https://github.com/dotnet/dotnet-api-docs/tree/main/snippets/visualbasic/VS_Snippets_Wpf/CustomControlNumericUpDown/visualbasic/customcontrollibrary).
 
  ]]></format>
         </remarks>
         <altmember cref="P:System.Windows.FrameworkElement.OverridesDefaultStyle" />
-        <related type="ExternalDocumentation" href="https://go.microsoft.com/fwlink/?LinkID=160025">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
+        <related type="ExternalDocumentation" href="https://github.com/dotnet/docs-desktop/tree/main/dotnet-desktop-guide/samples/snippets/csharp/VS_Snippets_Wpf/UserControlNumericUpDown/CSharp">NumericUpDown Custom Control with Theme and UI Automation Support Sample</related>
```

Comment:
These links should be altered too, perhaps to the local snippet, but not cross-repo.